app/models/publication.rb

```diff
@@ -141,6 +147,7 @@ def self.oa_workflow_states
   scope :activity_insight_oa_publication, -> { with_no_oa_locations.joins(:activity_insight_oa_files).where.not(activity_insight_oa_files: { location: nil }) }
   scope :doi_failed_verification, -> { activity_insight_oa_publication.where('doi_verified = false') }
   scope :needs_doi_verification, -> { activity_insight_oa_publication.where(doi_verified: nil).where(%{oa_workflow_state IS DISTINCT FROM 'automatic DOI verification pending'}) }
+  scope :needs_oa_metadata_search, -> { activity_insight_oa_publication.where(doi_verified: true).where(%{oa_workflow_state IS DISTINCT FROM 'no open access data found'}) }
```

Comment:
Depending on what we decide regarding my earlier comment, we might be doing away with `'no open access data found'` and either replacing it with something else, or not replacing it at all.

However, in any case, I think we do need to exclude publications with an `oa_workflow_state` of `'oa metadata search pending'`.